scenes/quests/story_quests/NO_EDIT/0_template_intro/template_intro_components/template_intro_image.png.import

```diff
@@ -3,28 +3,34 @@
 importer="texture"
 type="CompressedTexture2D"
 uid="uid://07k43id4c81h"
-path="res://.godot/imported/template_intro_image.png-401dee0e08a78cd4c45436c01fa38099.ctex"
+path="res://.godot/imported/template_intro_image.png-b2234a1131462d8ba4b3d04377c946a8.ctex"
 metadata={
 "vram_texture": false
 }
 
 [deps]
 
-source_file="res://scenes/quests/story_quests/template/0_template_intro/template_intro_components/template_intro_image.png"
-dest_files=["res://.godot/imported/template_intro_image.png-401dee0e08a78cd4c45436c01fa38099.ctex"]
+source_file="res://scenes/quests/story_quests/NO_EDIT/0_template_intro/template_intro_components/template_intro_image.png"
+dest_files=["res://.godot/imported/template_intro_image.png-b2234a1131462d8ba4b3d04377c946a8.ctex"]
 
 [params]
 
 compress/mode=0
 compress/high_quality=false
 compress/lossy_quality=0.7
+compress/uastc_level=0
+compress/rdo_quality_loss=0.0
```

Comment:
Did you write this PR with Godot 4.5? These properties are added by that version. When I try to edit the project in 4.4 it removes them again.